Dockerfile.node

```diff
@@ -1,5 +1,5 @@
 # syntax=docker.io/docker/dockerfile:1.3@sha256:42399d4635eddd7a9b8a24be879d2f9a930d0ed040a61324cfdf59ef1357b3b2
-FROM --platform=linux/amd64 docker.io/golang:1.17.5@sha256:90d1ab81f3d157ca649a9ff8d251691b810d95ea6023a03cdca139df58bca599 AS build
+FROM --platform=linux/amd64 docker.io/golang:1.8@sha256:fa71e1447cb0241324162a6c51297206928d755b16142eceec7b809af55061e5 AS build
```

Comment:
I assume we're going to wait for #1941 but if not, there's a typo here: "1.8" should be "1.18".

Reply:
Good catch!  I will rely on #1941 and revert updating golang.

Reply:
Actually I can still rely on golang 1.17 so will just remove 1.18 altogether.